src/CarbonSystemSolvers.jl

```diff
@@ -556,13 +1090,13 @@ end # end function
 end # module DirectCubicCarbonSolver
 
 # ----------------------------------------------------------------------------------
-
+# ## This should go in the testing suite, eventually.
```

Comment:
I see it is in the testing suite, right?
so let's remove from here?

Reply:
Done